<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProbeList</class>
 <widget class="QDialog" name="DialogProbeList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>探头</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="filter">
     <property name="placeholderText">
      <string>过滤...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>298</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="addProbe">
       <property name="text">
        <string>+</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Resources/Resources.qrc">
         <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeProbe">
       <property name="text">
        <string>-</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Resources/Resources.qrc">
         <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../resources/icons/Fatcow.qrc">
     <normaloff>:/fatcow/16x16/page_copy.png</normaloff>:/fatcow/16x16/page_copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../../resources/icons/Fatcow.qrc">
     <normaloff>:/fatcow/16x16/page_add.png</normaloff>:/fatcow/16x16/page_add.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/icons/Fatcow.qrc"/>
  <include location="../../../Resources/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogProbeList</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogProbeList</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
